apps/sudoku.html

```diff
@@ -58,6 +58,12 @@ <h1>Sudoko</h1>
 
       <div id="SudokuGrid"></div>
     </div>
+    <div class="row">
+      <div class="col">
+        <div id="numberButton"></div>
+      </div>
+    </div>
```

Comment:
So a few points here:
1. In Bootstrap there are 12 columns in a row. So each column div needs to have a screen width and a size. i.e. `col-xs-12` meaning for small screens and up this div should fill all 12 columns.
1. To get this to appear centrally you'll need to put it inside a container. Either wrap it inside a container div or use the one already on the page.